contact.html

```diff
@@ -0,0 +1,613 @@
+                                                        <img src="./assets/linkedin.svg" class="cmp-image__image"
+                                                            itemprop="contentUrl" data-cmp-hook-image="image"
+                                                            alt="LinkedIn">
+                                                    </a>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+          </div>
+          <!-- CONTACT -->
+          <div class="container responsivegrid contact-us aem-GridColumn aem-GridColumn--default--12">
+            <div id="contact-container" class="cmp-container">
+                <div class="aem-Grid aem-Grid--12 aem-Grid--default--12">
+
+                    <div class="container responsivegrid contact-content aem-GridColumn aem-GridColumn--default--none aem-GridColumn--default--5 aem-GridColumn--offset--default--0 aem-GridColumn--phone--none aem-GridColumn--phone--12 aem-GridColumn--offset--phone--0">
+                        <div class="cmp-container">
+                            <div class="contact-title">
+                                <div id="title-fca834a3a8" class="cmp-title">
+                                    <h1 class="cmp-title__text">Contact Us</h1>
+                                </div>
+                            </div>
+                            <div class="contact-subtext text base">
+                                <div data-cmp-is="adaptiveFormText" id="text-53cc58dc93" name="text_demo" data-cmp-adaptiveformcontainer-path="/content/core-components-examples/library/adaptive-form/text/jcr:content/root/responsivegrid/demo/component/guideContainer" class="cmp-adaptiveform-text">
+                                  <div tabindex="0">
+                                        <p>Want to know more? We would be happy to answer any questions you may have.</p>
+                                  </div>
+                                </div>
+                            </div>
+                            <div class="contact-subtitle">
+                                <div id="title-fca834a3a8" class="cmp-title">
+                                    <h6 class="cmp-title__text">LeaderImpact Global
+                                    </h6>
+                                </div>
+                            </div>
+                            <div class="container responsivegrid contact-text-button aem-GridColumn aem-GridColumn--default--none aem-GridColumn--default--5 aem-GridColumn--offset--default--0 aem-GridColumn--phone--none aem-GridColumn--phone--12 aem-GridColumn--offset--phone--0">
```

Comment:
@Prystal254 I believe this needs to be wrapped in a `<div class="aem-Grid aem-Grid--12 aem-Grid--default--12">`.